demos/sev-demo/README.md

```diff
@@ -0,0 +1,136 @@
+# CCv0 with AMD SEV
+
+[Demo Video](https://youtu.be/SXZ-hzhjahU)
+
+AMD SEV is already enabled in upstream Kata. SEV can also be used with CCv0. Although most of the code needed for CCv0 with SEV is upstream in CCv0, currently some manual configuration is required. This is described here. Some of these steps will soon be automated. These instructions assume that you already have CCv0 setup and that you have a node where you can launch SEV VMs.
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate firmware binary is available in this directory. You can also build from source. To do so, follow the standard OVMF build procedure, but use `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc` rather than the OVMF DSC file. The AMDSEV build currently expects to use Grub for reasons unrelated to Kata. To bypass this, you should comment out line 28 of `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc`. If the build process complains about a missing `grub.efi` file, you can create a dummy file with that name and try again. This dependency may be adjusted in the future and we are hoping to provide a prebuilt version of the appropriate firmware.
```

Comment:
Not related to this PR directly, but I would suggest to host an OVMF branch with the needed (one-liner iiuc) patch for that.

Reply:
I completely redid this section. It still has the potential to rot, but this part of OVMF doesn't really see much development.